### DISCUSSION TITLE
I need to comment the parameters in the URL address

### DISCUSSION THREAD

**Discussion by hanbinloop** [pydantic/pydantic#4603] [Question] [answered]
Here's my view definition:
![image](https://user-images.githubusercontent.com/16644654/194813088-a9a662e4-5d9f-48e0-8ae9-47b59eeadb3a.png)
This is openapi. Json:
![image](https://user-images.githubusercontent.com/16644654/194813138-4a6622d4-d946-44bb-8767-48ec697106fb.png)
I need to change the title in the schema to Chinese

**Comment by samuelcolvin** [comment, marked as answer]
Please don't use screen shots for code: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code

To set the title I believe you can set `title` in config https://pydantic-docs.helpmanual.io/usage/model_config/#options
